src/auth.rs

```diff
@@ -0,0 +1,141 @@
+use crate::openapi::apis::configuration::Configuration;
+use crate::openapi::apis::{authenticate_api, transactions_api};
+use crate::Error;
+
+pub struct Auth {
+    pub(crate) configuration: Configuration,
```

Comment:
This looks like it'd make the configuration of the struct public within the crate.
Where are the internal crate modules accessing this field?

Reply:
ah sorry this was leftover from when i was trying something earlier [9895501](https://github.com/passageidentity/passage-flex-rust/pull/30/commits/98955014feda994b816cbe8c2d51e99a7fc987f9)

Reply:
to circle back, the `(crate)` modifier was necessary to keep compatibility with the test mocks via the `set_server_url` method in the base class. dropped the above commit in a rebase to keep the modifier